pkg/support/dump_others.go

```diff
@@ -59,6 +63,40 @@ func (d *agentDumper) dumpIPTables(basedir string) error {
 	return writeFile(d.fs, filepath.Join(basedir, "iptables"), "iptables", data)
 }
 
+func (d *agentDumper) dumpNFTables(basedir string) error {
+	c, err := nftables.New(d.v4Enabled, d.v6Enabled)
+	if err != nil {
+		klog.Warningf("Skipping nftables dump, client init failed: %v", err)
+		return nil
+	}
+
+	if d.v4Enabled && c.IPv4 != nil {
+		output, err := c.IPv4.DumpTable(context.TODO(), "antrea")
```

Comment:
I didn't find the method `DumpTable` in the interface from nftables lib https://github.com/hongliangl/knftables/blob/e4307300abb50f2d9e7a492c474fe046d777f87f/nftables.go#L30.

Please use the appropriate methods to dump the table.